source/MapOutfitterPanel.h

```diff
@@ -51,9 +51,7 @@ class MapOutfitterPanel : public MapSalesPanel {
 
 	virtual void DrawItems() override;
 
-
-private:
-	void Init();
+	virtual void Init() override;
```

Comment:
Why did you change stuff with the init methods? Seems unrelated to me?

Reply:
It's so I can call `Init()` from `MapSalesPanel::KeyDown()` to recompile the catalog when the 'l' key is pressed.

Reply:
oh right, that's ok I think since it doesnt happen that often and we'd do the same thing anyway if we did a separate method